test/helper.go

```diff
@@ -23,7 +23,7 @@ func testClient() *client.APIClient {
 	configuration.HTTPClient = &http.Client{
 		Transport: &debugTransport{
 			Transport:   http.DefaultTransport,
-			EnableDebug: true,
+			EnableDebug: false,
```

Comment:
Weirdly enough, this makes the tests so much more readable when something goes wrong since the error output isn't hidden by the output of several hundred HTTP calls

If you hate it, here's a nice convenient way of rolling it back!

```suggestion
			EnableDebug: true,
```